### ISSUE TITLE
cant select alt text - desktop firefox or android

### ISSUE THREAD

**Issue by monicaellerose** [bluesky-social/social-app#1548]
cannot select alt text on android or firefox (to copy/paste )

tried multiple times

ideal situation, to be able to select alt text on photos

current behavior of mouse when hovering over alt text

![PXL_20230927_041741015 MP](https://github.com/bluesky-social/social-app/assets/66647238/672c89e0-7702-4a1c-8f27-468e46100ca7)

**Comment by hkolbeck** [comment]
I raised this with @pfrazee 2 weeks ago (https://bsky.app/profile/hannah.not-ok.computer/post/3k7cb4zigaj2o), apologies for not making a ticket. Currently the alt text is wrapped in a button, changing that to a div instead allows copying and I was able to confirm still handles expand/collapse.

**Comment by monicaellerose** [comment]
@hkolbeck I made this as a new issue, as I wasnt sure if they tried to fix it in v 1.51  

https://bsky.app/profile/bsky.app/post/3kadh65z2el2r

**Comment by pfrazee** [comment]
Got a fix for this waiting for review https://github.com/bluesky-social/social-app/commit/76aec80fb8e745ed2b3af3d356e238466042d892
